cmd/minikube/cmd/config/config.go

```diff
@@ -143,12 +148,12 @@ func ReadConfig() (MinikubeConfig, error) {
 func WriteConfig(m MinikubeConfig) error {
 	f, err := os.Create(constants.ConfigFile)
 	if err != nil {
-		return fmt.Errorf("Could not open file %s: %s", constants.ConfigFile, err)
+		return fmt.Errorf("Cannot open file %s: %s", constants.ConfigFile, err)
 	}
 	defer f.Close()
 	err = encode(f, m)
 	if err != nil {
-		return fmt.Errorf("Error encoding config %s: %s", constants.ConfigFile, err)
+		return fmt.Errorf("Cannot write config %s: %s", constants.ConfigFile, err)
```

Comment:
This is encoding from the Go _MinikubeConfig_ data structure into a JSON document. So yes, formats are changed here.